src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +512,26 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null, or empty
+     * or contains only whitespace characters.
+     * More efficient that calling (string).trim().length() == 0
+     * or (string).trim().isEmpty()
+     * because it does not create temporary string whereas trim() does
+     * @param value A string to check
+     * @return true if the value does not contain any character except whitespace.
```

Comment:
yes, functionality is the same - http://commons.apache.org/proper/commons-lang/apidocs/src-html/org/apache/commons/lang3/StringUtils.html#line.295

I would vote to name it the same way as Apache - `isBlank` and even mention that in javadoc. `isBlank` is very known method in web , so reusage of its name will be good (as functionality is the same).
@rnveach , do you agree ? please share your final vote and let close this PR.
@Kietzmann , @Vladlis please share share your vote too.

a) isBlank
b) isOnlyWhitespacesOrEmpty
c) isEmptyOrOnlyWhitespaces
d) isNullOrBlank
e) isNullOrEmptyOrWhitespacesOnly
f) your variant

>  I would opt for the long name rather than for the name which is lying to me.

I like this principle too , but do not be very fanatic with it. You can not put all details of implementation in name of method. It is not possible. Verification of null is almost never present in name(it is present only when it is critical... isNullOrEmpty, ..... ). It is better to keep code readable (very uncertain criteria :) ). Long names is not always do good job.